invenio/modules/search/views/search.py

```diff
@@ -462,7 +462,7 @@ def search(collection, p, of, ot, so, rm):
 
     ctx = dict(
         facets=facets.get_facets_config(collection, qid),
-        records=len(get_current_user_records_that_can_be_displayed(qid)),
+        records=len(recids),
```

Comment:
@tiborsimko I don't remember why it was there. Maybe it was copied&pasted from legacy code and some parts got refactored later. IMHO we can use `recids` directly in this case (facets should stay as they are).